components/restservice/scripts/create.py

```diff
@@ -60,8 +60,9 @@ def install_optional(rest_venv):
         ctx.logger.info('Installing REST Service...')
         utils.install_python_package('/tmp/rest-service', rest_venv)
         ctx.logger.info('Deploying Required Manager Resources...')
-        utils.move(
+        utils.copy(
```

Comment:
move is not idempotent. We should consider changing the move impl since it will throw an error if files are already in place as in the above example.